### ISSUE TITLE
Getting an error when simulate

### ISSUE THREAD

**Issue by planetahuevo** [sebastianfeldmann/phpbu#113]
I am getting this error on a simulate. 
Testing right now the backup to see if I have the same issue.

`Exception 'RuntimeException' with message 'SplFileInfo::getMTime(): stat failed for /****/filesbackup.xxx' in phar:///usr/local/bin/phpbu/Backup/File.php:94`

**Comment by planetahuevo** [comment]
Same backup without simulate works fine.

**Comment by sebastianfeldmann** [comment]
What kind of backup is causing the problem?
```xml
<backup type="???">
```
Can you please provide the backup configuration?

```xml
<backup>
...
</backup>
```

**Comment by sebastianfeldmann** [comment]
Would be great if you could remove all extra configuration `check`, `crypt`, `sync` and `cleanup` to narrow down which simulation causes the problem

**Comment by planetahuevo** [comment]
I was able to debug the problem to the cleanup code. 
I have this:
```
<!-- cleanup capacity: clean all -->
	<cleanup type="capacity">
	  <option name="size" value="0B"/>
	  <option name="deleteTarget" value="true"/>
	</cleanup>
```

**Comment by sebastianfeldmann** [comment]
Thanks for taking the time to debug this
I will have a look at the Cleanup simulation code and hopefully release a fix soon

**Comment by planetahuevo** [comment]
Is the least I can do. Happy to help on what I can. :)

**Comment by planetahuevo** [comment]
One more thing, I only have that cleanup option with a tar backup. 
The quantity I use for the db backups works fine.

**Comment by sebastianfeldmann** [comment]
I just released version 5.0.4 with a fix for this issue ;)
